src/contracts/deploymentHelper/DeploymentHelper.sol

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
```

Comment:
I'd rename the folder from `deploymentHelper` to just `deployment`, since it's implicit that it's a helper. Note that there will be another contract to manage the deployment soon.
I would even consider having any root contract at the root of the repo instead of in a subfolder, so it's clear what the entry points are, but since these will likely not be audited in time and they don't directly affect the functionality of the token, I'm ok with keeping them in `deployment`.